src/VisualStudio/Core/Def/Implementation/FindReferences/StreamingFindUsagesPresenter.cs

```diff
@@ -275,5 +289,58 @@ protected static (Guid, string projectName, string? projectFlavor) GetGuidAndPro
 
             return (guid, projectName, projectFlavor);
         }
+
+        private void RemoveExistingInfoBar()
+        {
+            Contract.ThrowIfFalse(this.ThreadingContext.HasMainThread);
+
+            var infoBar = _infoBar;
+            _infoBar = null;
+            if (infoBar != null)
+            {
+                var infoBarHost = GetInfoBarHost();
+                infoBarHost?.RemoveInfoBar(infoBar);
+            }
+        }
+
+        private IVsInfoBarHost? GetInfoBarHost()
+        {
+            Contract.ThrowIfFalse(this.ThreadingContext.HasMainThread);
+
+            // Guid of the FindRefs window.  Defined here:
+            // https://devdiv.visualstudio.com/DevDiv/_git/VS?path=/src/env/ErrorList/Pkg/Guids.cs&version=GBmain&line=24
+            var guid = new Guid("a80febb4-e7e0-4147-b476-21aaf2453969");
+
+            if (_serviceProvider.GetService(typeof(SVsUIShell)) is not IVsUIShell uiShell ||
+                uiShell.FindToolWindow((uint)__VSFINDTOOLWIN.FTW_fFindFirst, ref guid, out var windowFrame) != VSConstants.S_OK ||
+                windowFrame == null ||
+                windowFrame.GetProperty((int)__VSFPROPID7.VSFPROPID_InfoBarHost, out var infoBarHostObj) != VSConstants.S_OK ||
+                infoBarHostObj is not IVsInfoBarHost infoBarHost)
+            {
+                return null;
+            }
+
+            return infoBarHost;
+        }
+
+        private async Task ReportInformationalMessageAsync(string message, CancellationToken cancellationToken)
+        {
+            await this.ThreadingContext.JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
+            RemoveExistingInfoBar();
+
+            if (_serviceProvider.GetService(typeof(SVsInfoBarUIFactory)) is not IVsInfoBarUIFactory factory)
+                return;
+
+            var infoBarHost = GetInfoBarHost();
+            if (infoBarHost == null)
+                return;
+
+            _infoBar = factory.CreateInfoBar(new InfoBarModel(
+                message,
+                KnownMonikers.StatusInformation,
+                isCloseButtonVisible: false));
+
+            infoBarHost?.AddInfoBar(_infoBar);
```

Comment:
Null checked above already.

```suggestion
            infoBarHost.AddInfoBar(_infoBar);
```

Reply:
Indeed